tests/e2e/config/wdio.ios.rdc.conf.js

```diff
@@ -0,0 +1,31 @@
+const { config} = require('./wdio.rdc.shared');
+
+// ============
+// Capabilities
+// ============
+// For all capabilities please check
+// http://appium.io/docs/en/writing-running-appium/caps/#general-capabilities
+config.capabilities = [
+  {
+    deviceName: 'iPhone 8',
+    // The reference to the app
+    testobject_app_id: '1',
+    // The api key that has a reference to the app-project in the TO cloud
+    testobject_api_key: process.env.SAUCE_RDC_EU_ACCESS_KEY_IOS,
+    // The name of the test for in the cloud
+    testobject_test_name: 'sample-app-mobile',
+    // Some default settings
+    // You can find more info in the TO Appium Basic Setup section
+    platformName: 'iOS',
```

Comment:
No platform version here - do we know what version of iOS will be present during testing?

Reply:
For now we just pick a iPhone 8 with device in the cloud with dynamic allocation.